src/ui/public/visualize/visualize.less

```diff
@@ -15,11 +15,16 @@ visualize {
   .vis-container {
     display: flex;
     flex-direction: row;
-
-    flex: 1 0;
     overflow: auto;
     -webkit-transition: opacity 0.01s;
     transition: opacity 0.01s;
+    flex: 1 1 0;
+
+    // IE11 hack
+    // https://github.com/philipwalton/flexbugs#4-flex-shorthand-declarations-with-unitless-flex-basis-values-are-ignored
```

Comment:
I know this is kind of stupidly over-the-top, but because this hack is needed to fix a _Firefox_ bug, I think we should elaborate, e.g.:

```less
// Normally we would just set flex: 1 1 0%, which works as expected in IE11.
// Unfortunately, a recent bug in Firefox causes this rule to be ignored, so we
// have to use an IE-specific hack instead.
```